src/core/why3_tactic.ml

```diff
@@ -4,7 +4,9 @@ open Console
 open Terms
 open Extra
 open Timed
-open Scope
+open Print
+
+module Builtin = Sig_state.Builtin
```

Comment:
Please avoid module aliases, rather open the module locally using `let open Sig_state.Builtin`.